Tests/lib/taskRunner.ts

```diff
@@ -18,7 +18,7 @@ function debug(message) {
 }
 
 export class TaskRunner extends events.EventEmitter {
-	constructor(name: string, normalizeSlashes?: boolean, ignoreTempPathsInResponse?: boolean) {
+	constructor(name: string, normalizeSlashes?: boolean, ignoreTempPathsInResponse?: boolean, wildcardAccepted?: boolean) {
```

Comment:
Would other tasks also require change because of the change in constructor and they would default to false for wildcaredaccepted